.env

```diff
@@ -5,4 +5,4 @@ POSTGRES_USER=zammad
 REDIS_URL=redis://zammad-redis:6379
 RESTART=always
 # don't forget to add the minus before the version
-VERSION=-5.0.3-7
+VERSION=-5.0.3-13
```

Comment:
That tag is not yet available. You can't use it for that reason.

Reply:
Understood, thank you!